tests/basic/misc_test.go

```diff
@@ -1784,4 +1803,47 @@ var _ = Describe("{KubeClusterRestart}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
-})
\ No newline at end of file
+})
+
+var _ = Describe("{VerifyNoPxRestartDueToPxPodStop}", func() {
+	JustBeforeEach(func() {
+		// https://purestorage.atlassian.net/browse/PTX-24859
+		StartTorpedoTest("VerifyNoPxRestartDueToPxPodStop", "Verify that px serivce remain up even if px pod got deleted ", nil, 87311294)
+	})
+
+	It("has to setup, validate and teardown apps", func() {
+		// Get uptime for px service on each node
+		stepLog = "Getting PID of Px process before and after restarting PX pods on all the nodes"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			processPid, err := GetPxPIDMap(node.GetStorageDriverNodes())
+			if err != nil {
+				log.FailOnError(err, "Failed while getting PID of PX process")
+			}
+
+			namespace, err := Inst().S.GetPortworxNamespace()
+			log.FailOnError(err, "We have not Px pods in any of the namespace")
+
+			//Deleting px pods from all the node
+			err = DeletePXPods(namespace)
+			log.FailOnError(err, "We have come across error while deleting PX pod ")
+
+			//Capturing PID of PX after stopping PX pods
+			processPidPostRestart, err := GetPxPIDMap(node.GetStorageDriverNodes())
+			if err != nil {
+				log.FailOnError(err, "Failed while getting PID of PX process")
+			}
+			log.Infof("Process IDs for px after stopping portworx pod  %s", processPidPostRestart)
+
+			//Verify PID before and after for PX process
+			for nodeId, beforePID := range processPid {
+				afterPID, _ := processPidPostRestart[nodeId]
+				dash.VerifyFatal(beforePID, afterPID, fmt.Sprintf("Process ID of PX process before and after PX pod restart is different"))
```

Comment:
change this to fmt.Sprintf("Validate Process ID of PX process before and after PX pod restart")